components/create-post-page/TextbookUploaded.tsx

```diff
@@ -19,35 +10,27 @@ interface Props {
 
 export default function TextbookUploaded({ post }: Props) {
   return (
-    <Container maxWidth="container.md">
-      <VStack spacing="3">
-        <PostCard post={post} isLinkActive={true} />
-        <Container textAlign="left">
-          <Text>
-            Your textbook post is now live! Make sure to check your messages
-            regularly for messages from potential buyers. You can modify your
-            post or take it down at any time by going to your account page.
-          </Text>
-        </Container>
-        <Container>
-          <HStack direction="row" justify="end">
-            <Box />
-            <Spacer />
-            <Box>
-              <Link href="/account" passHref>
-                <Button
-                  as="a"
-                  colorScheme="teal"
-                  variant="ghost"
-                  rightIcon={<Icon as={UserIcon} />}
-                >
-                  Go to my account page
-                </Button>
-              </Link>
-            </Box>
-          </HStack>
-        </Container>
+    <Box maxWidth="md" margin="auto">
+      <PostCard post={post} isLinkActive={true} />
+      <VStack mt="6" spacing="3">
+        <Text>
+          Make sure to regularly check for messages from potential buyers.
+        </Text>
+        <Text>
+          You can modify your post or take it down at any time by going to your{" "}
+          <Link href="/account" passHref>
```

Comment:
Consider using the avatar icon instead. It is more descriptive than the link icon.